### ISSUE TITLE
[4.0] [Bug] SimpleMDE breaks when multiple simplemde fields are set and applies to the first textbox on page

### ISSUE THREAD

**Issue by SamSebastien** [Laravel-Backpack/CRUD#2089]
# Bug report
SimpleMDE breaks when multiple simplemde fields and applies to the first textbox on page

### What I did:
Used both single and multiple `'type' => 'simplemde'` fields.

### What I expected to happen:
SimpleMDE to deploy on each tagged field.

### What happened:
SimpleMDE deploys once and attaches to the first textarea field on the page, even if that is not set to be a simplemde field. Additionally, if multiple simplemde fields are set in the controller, it loops and creates multiple copies of the first instantiation. 

### What I've already tried to fix it:
Tried to override the field template, but so far no success.

### Backpack, Laravel, PHP, DB version:
Backpack 4.0, Laravel 5.8

Controller:
```php
$this->crud->addField([
	  'label' => 'Summary',
	  'name'  => 'summary',
	  'type'  => 'textarea'
  ]);
$this->crud->addField([
	  'label' => "Description",
	  'name'  => 'description',
	  'type'  => 'simplemde',
  ]);
$this->crud->addField([
	  'label' => "Profile",
	  'name'  => 'profile',
	  'type'  => 'simplemde'
  ]);
```

Screenshot:
![image](https://user-images.githubusercontent.com/9647970/65811526-195cd300-e1fd-11e9-83ee-a5106941708b.png)

**Comment by SamSebastien** [comment]
This seems to solve it:

https://github.com/Laravel-Backpack/CRUD/blob/085fe9ff4a0c325087a88914bc68681db5dcac4e/src/resources/views/crud/fields/simplemde.blade.php#L51

```js
var configurationObject = {
	element: $(elementId)[0],
};
```
should be:

```js
var configurationObject = {
	element: $('#' + elementId)[0],
};
```

**Comment by tabacitu** [comment]
Ouch! Don't know how we missed that in testing - thank you for the detailed bug report @SamSebastien , and especially for the fix :-) 

I can confirm the bug existed, and your fix did a great job. I've applied it on the ```master``` branch and it'll be available with a ```composer update``` in a few hours, after I bundle it with a few more bug fixes - make sure you get version 4.0.4 or higher.

Sorry for causing you issues. Please mention this issue if/when you apply for a free project license, or email me if you'd like a discount to an unlimited license. Again - sorry for the trouble, this is something that should've been easy to catch.

Cheers!
